ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -163,6 +288,13 @@
   @Default("false")
   public final boolean cosmosDirectHttps;
 
+  /**
+   * Azure storage account info.
+   */
+  @Config(AZURE_STORAGE_ACCOUNT_INFO)
+  @Default("")
+  public final String getAzureStorageAccountInfo;
```

Comment:
We should rename this to azureStorageAccountInfo?

Reply:
Done